Modules/Core/include/selxAccepting.hxx

```diff
@@ -127,8 +127,8 @@ template< typename FirstInterface, typename ... RestInterfaces >
 bool
 Accepting< FirstInterface, RestInterfaces ... >::AreAllAccepted()
 {
-  InterfaceAcceptor< FirstInterface > * acceptIF = ( this );
-  if( acceptIF->isSet() == false )
+  // The interface acceptor has stored the last accepted interface, which is empty if no connection was made.
+  if (!this->InterfaceAcceptor< FirstInterface >::GetAccepted())
```

Comment:
Should this be GetPointer()? It does not read well and I wouldn't know a pointer was returned if we hadnt discussed this in person (I thought it was a boolean)

Reply:
Not sure what you mean. In general I would not be in favour of using GetPointer() on shared_ptrs, since there is no need to expose raw pointers explictly here.
It would indeed read better if the check performs an "is_empty" on the shared_ptr from GetAccepted(), but there is no such method in std. Implicit casting of (shared) pointers to booleans is standard practice I would say.

Reply:
Nevermind the suggestion, we can always come up with something better. What I mean is "GetAccepted" does not read well. It is not clear at first glance what "GetAccepted" is. GetAcceptedWhat? A word should be appended to "GetAccepted"